pkg/kubectl/cmd/apply.go

```diff
@@ -123,6 +128,27 @@ func validatePruneAll(prune, all bool, selector string) error {
 	return nil
 }
 
+func validatePruneResources(pruneResources *[]pruneResource, gvks []string) error {
```

Comment:
Thanks for the comment. I changed the name to `parsePruneResources` and put it into RunApply().